mailpoet/lib/Migrator/Repository.php

```diff
@@ -55,10 +55,15 @@ public function create(string $level): array {
    * @return array<array{level: string, name: string}>
    */
   public function loadAll(): array {
-    return array_merge(
+    $migrations = array_merge(
       $this->loadForLevel(self::MIGRATIONS_LEVEL_DB),
       $this->loadForLevel(self::MIGRATIONS_LEVEL_APP)
     );
+    $duplicateNames = array_diff_assoc(array_column($migrations, 'name'), array_unique(array_column($migrations, 'name')));
```

Comment:
Not important, but `array_column($migrations, 'name')` could be saved to a `$names` var first, since it's called twice.